src/main/java/org/folio/marccat/resources/HeaderTypeAPI.java

```diff
@@ -29,4 +34,21 @@ public HeadingTypeCollection getHeadingTypes(
       return null;
     }, tenant, okapiUrl, configurator);
   }
+
+  @GetMapping("/auth-header-types")
+  public HeadingTypeCollection getAuthorityHeadingTypes(@RequestParam final String code,
```

Comment:
What is `code` here? I don't see it used in this method nor do I see it in the RAML.

Reply:
I have used it.

Reply:
Could you explain how `code` is used? Like I said, I don't see it referenced in this method. There is a `setCode` below, but that uses a constant. How is this query param used?

Reply:
I have changed the function to control the code value.
`      HeadingType headingType = new HeadingType();
      if (code.equals(Global.AUTHORITY_MATERIAL_DESCRIPTION_DESCRIPTION))
        headingType.setCode(Global.AUTHORITY_MATERIAL_DESCRIPTION_HEADER_TYPE);
      headingType.setDescription(code);
`